client/src/main/java/org/spine3/base/Commands.java

```diff
@@ -130,7 +129,7 @@ public static CommandId getId(Command command) {
             public boolean apply(@Nullable Command request) {
                 checkNotNull(request);
                 final Timestamp timestamp = getTimestamp(request);
-                return Timestamps.isAfter(timestamp, from);
+                return Timestamps.isLater(timestamp, from);
```

Comment:
The current name is in-line with [naming in new date/time API in Java](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html#isAfter-java.time.chrono.ChronoLocalDate-).